src/stackComponent/src/Stack.js

```diff
@@ -0,0 +1,49 @@
+import React, { memo, forwardRef } from 'react'
+import PropTypes from 'prop-types'
+import { spacing, dimensions, position, layout } from 'ui-box'
+import { Pane } from '../../layers'
+
+const Stack = memo(
```

Comment:
We _could_ introduce a breaking change and export this as `Stack`. I think we'd have to rename our existing zindex `Stack` component as something else – maybe `Layer`?